package.json

```diff
@@ -4,52 +4,52 @@
   "description": "Javascript Answers Programming Interface",
   "main": "gulpfile.js",
   "dependencies": {
-    "@yext/rtf-converter": "^1.0.0",
+    "@yext/rtf-converter": "^1.1.1",
```

Comment:
I think updating our npm deps would be great, but it should be a separate PR. I believe our stance is we need to be careful to test that nothing breaks when we update a dependency (@tmeyer2115 correct me if I'm wrong). probably stuff used only in testing like testcafe and jest can be updated whenever,  but things that affect the actual bundle like handlebars have a bit of risk